### DISCUSSION TITLE
Key mappings for mod

### DISCUSSION THREAD

**Discussion by AlbertoCentonze** [qtile/qtile#2880] [Q&A] [answered]
Hello there!
I've not been able to find anywhere a list with all the names of the keys and their corresponding strings (like "mod4" for super). I've spent a whole hour looking inside the documentation and in other people's dotfiles and still I couldn't find anything.
Is there an actuals section about this in the docs? If that's not the case I would love to add it!
Btw I really love this project, especially because Python is such an accessible language that I can write my own extensions and configs in a familiar yet powerful way!

**Comment by hamensman** [comment, marked as answer]
Download and read the `xkeysyms.py` here for all the key names: https://github.com/qtile/qtile/blob/master/libqtile/backend/x11/xkeysyms.py

As for the mod keys, I declare them in my config like this:
```
mod = "mod4"
mod1 = "alt"
mod2 = "control"
```

**Reply by AlbertoCentonze** [reply]
Thanks a lot for your prompt reply! By the way, I still think that this should be referenced in the docs because it really drove me crazy. Can I make a pull request? In what part of the wiki should I mention the existence of this file?

**Reply by m-col** [reply]
I suggest adding a link somewhere in the docs (as opposed to the wiki) near where the keybindings are documented.

**Reply by AlbertoCentonze** [reply]
Actually, I've found what I've been looking for [here](http://docs.qtile.org/en/latest/manual/faq.html#how-do-i-know-which-modifier-specification-maps-to-which-key). Sorry for wasting your time. I really love qtile and I can't wait to contribute!
